recipes/freebayes/meta.yaml

```diff
@@ -20,18 +21,23 @@ requirements:
     - bzip2
     - python
     - xz
+    
   run:
     - zlib
     - bzip2
     - python
     - htslib
     - parallel
     - xz
+    - bc
+    - samtools
+    - meson
```

Comment:
meson and nija should probably be in the host section, isn't it?

Reply:
https://github.com/bioconda/bioconda-recipes/blob/master/recipes/pbmm2/meta.yaml has in in the build section, but yes I think you are right.  I'm a few months rusty doing this.